### ISSUE TITLE
Set Learning Rate

### ISSUE THREAD

**Issue by john2king** [bgshih/crnn#28]
I can not find any config about the learning rate in config.lua,and the current learning rate also can not be display at each iteration during training.Do you have any idea about it?
Thanks!

**Comment by Suyuanhang** [comment]
@john2king @bgshih When I was trying to train the network using my own dataset, I found the training loss is always infinite, when I'm trying to config the learning rate, I encounter the same problem as above.

@bgshih There should be no theoretical limit to implement the learning rate, can you let us know where it is?
Thank you

**Comment by misssprite** [comment]
@Suyuanhang 
The default optimization algorithm used in config.lua is [`adadelta`](https://github.com/torch/optim/blob/master/doc/algos.md#optim.adadelta) , which adjust learning rated automatically. You can try other algorithms like sgd with a learning rate.
Adadelta is good enough for me.

**Comment by tolga-b** [comment]
in config.lua in function getConfig change the following lines to:
```
...
optimMethod = optim.sgd,
optimConfig = {learningRate = 1e-3, momentum = 0.9},
...
```
check https://github.com/torch/optim/blob/master/sgd.lua for more details.
